### ISSUE TITLE
Extension activation failed

### ISSUE THREAD

**Issue by mackellardrew** [microsoft/vscode-jupyter#6283]
## Environment data

-   VS Code version: 1.53.2
-   Jupyter Extension version (available under the Extensions sidebar): 2021.2.603412351
-   Python Extension version (available under the Extensions sidebar): 2021.2.633441544
-   OS (Windows | Mac | Linux distro) and version: Windows 10 Enterprise v10.0.19042
-   Python and/or Anaconda version: Python 3.8.5; conda 4.10.1
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Jupyter server running: Local

## Expected behaviour

Jupyter notebooks load normally

## Actual behaviour

An window pops up that says 

"Extension activation failed, run the 'Developer: Toggle Developer Tools' command for more information."

Notebook renders as standard JSON.  Opening the Developer tools window shows logs; attached.

## Steps to reproduce:

Just opening a Jupyter notebook with this extension version seems to do it.

I should note that I've been using VS Code with Jupyter productively for perhaps a year; most of it with probably these same versions extensions (I don't recall actively changing the configuration recently); this issue is only encountered recently.  I typically use with either remote or WSL servers; this same error appears regardless of which server type is used: native Windows, WSL, or on a cloud instance.

Also, I had been using these older versions of VS Code and, necessarily, both extensions, on a work device over which I lack admin privileges.  Once I was able to confirm that the newest versions of these tools, namely:

VS Code v1.57.0
Python extension v2021.5.926500501
Jupyter extension v2021.6.999230701

work on a personal PC over which I do have complete control, I was also able to install the same versions of the app from the standalone (zip) distribution on all devices, so I consider the issue resolved and have no personal stake in seeing the issue resolved for legacy versions of the tools; I also understand that dev time is precious and you may just want to close this immediately.  I just thought that filing it out of historical interest/FYI might be a good thing to do.

## Logs

[20210614_vscode_jupyter_extension_win.log](https://github.com/microsoft/vscode-jupyter/files/6651658/20210614_vscode_jupyter_extension_win.log)

**Comment by IanMatthewHuff** [comment]
Hey @mackellardrew . Thanks for filing. I would say that our support might be limited here with what we can do for that old a version. The one thing that does surprise me is that you mentioned that it suddenly stopped working? In general if you had a working set of VS Code + Extension then it would continue to work together, even if breaking changes continue to come in. Did something update on that machine?

**Comment by mackellardrew** [comment]
Yes, I think in retrospect that something must have changed with the configuration, despite my not intentionally changing any versions.  But I did confirm that attempting to install those exact versions (of VS Code and Python and Jupyter extensions) sufficed to reproduce the same error message on my other laptop, so it appears to me to be a stable outcome of combining those packages.  Which would mean that I would've expected the same behavior from my work machine all along had nothing been altered behind the scenes.

The change in the support for notebooks in fact appears to have occurred right after I had installed VS code for the first time on my personal laptop, and I do use sync settings across my installations, so perhaps the addition of a new install on a different machine introduced some kind of change beyond the version numbers of the app & extensions that would have caused the failure.  My personal laptop should be the latest build of Windows 10, whereas my work stages rollout of new OS versions such that we're not as up-to-date; perhaps that also might have had some effect in enforcing some kind of compromise in the settings that would be compatible across all installations?  I think the issue may be knotty and nuanced, so I understand if it's not worth pursuing.  But if you want any additional info about my environment on the other laptop than was cited in the original report, I'd be happy to send that along.
